src/editor/sidebar/index.js

```diff
@@ -229,7 +229,7 @@ class Sidebar extends Component {
 					<Button
 						isLink
 						onClick={ () => this.updateSender( senderName, senderEmail ) }
-						disabled={ inFlight }
+						disabled={ inFlight || senderEmail.length ? ! hasValidEmail( senderEmail ) : false }
```

Comment:
Currently the button is enabled during "in flight." 

```suggestion
						disabled={
							inFlight ||
							! senderEmail.length ||
							! senderName.length ||
							! hasValidEmail( senderEmail )
						}
```

Reply:
Fixed: a31550d789321e5d78752b776e27e4d370576937